packages/inspector/src/lib/data-layer/host/utils/composite-dirty.ts

```diff
@@ -171,6 +163,9 @@ export async function compositeAndDirty(
         return;
       }
       dirty = DirtyEnum.DirtyAndDump;
+      const shouldDump =
+        getInspectorPreferences().autosaveEnabled && dirty === DirtyEnum.DirtyAndDump;
```

Comment:
at this point, `dirty === DirtyEnum.DirtyAndDump` will always be true

Reply:
yeah, leaving it there just for hard reference